includes/widgets/image-box.php

```diff
@@ -115,7 +115,7 @@ protected function register_controls() {
 		$this->add_control(
 			'title_text',
 			[
-				'label' => esc_html__( 'Title & Description', 'elementor' ),
+				'label' => esc_html__( 'Title', 'elementor' ),
```

Comment:
This control labeled "Title", the second one should be labeled "Description".

Reply:
Done.